iset.mm

```diff
@@ -98980,9 +98980,12 @@ equinumerous to omega (the set of finite ordinal numbers).  (Contributed
 $)
 
   $c seq $.
-  $( Extend class notation with recursive sequence builder. $)
+  $( Extend class notation with four argument recursive sequence builder. $)
```

Comment:
four-argument

Reply:
This could turn into one of those epic style guide things, but I will add the hyphen here and hopefully save us that trouble.

Reply:
Is that a thousand-year-old debate ? ;) (in the present case, this was more about consistency, and triggered by the other comment)

Reply:
> Is that a thousand-year-old debate ? ;) (in the present case, this was more about consistency, and triggered by the other comment)

LOL, clever use of hyphens in "thousand-year-old"